core/chains/evm/txmgr/txmgr_test.go

```diff
@@ -758,4 +758,5 @@ func TestTxm_Reset(t *testing.T) {
 		require.NoError(t, err)
 		assert.Equal(t, 0, count)
 	})
+	assert.NoError(t, txm.Close())
```

Comment:
What about calling from `defer` or `t.Cleanup`, just after `txmgr.NewTxm()`?

Reply:
yep, you are right. put into defer()